include/nbl/ext/FullScreenTriangle/FullScreenTriangle.h

```diff
@@ -96,11 +102,16 @@ namespace nbl
 				records.
 			*/
 
-			inline bool recordDrawCalls(video::IGPUCommandBuffer* commandBuffer)
+			inline bool recordDrawCalls(core::smart_refctd_ptr<nbl::video::IGPUGraphicsPipeline> gpuGraphicsPipeline, core::smart_refctd_ptr<nbl::video::ISwapchain> swapchain, video::IGPUCommandBuffer* commandBuffer)
 			{
 				_NBL_STATIC_INLINE_CONSTEXPR auto VERTEX_COUNT = 3;
 				_NBL_STATIC_INLINE_CONSTEXPR auto INSTANCE_COUNT = 1;
 
+				auto layout = gpuGraphicsPipeline->getRenderpassIndependentPipeline()->getLayout();
+				const asset::SPushConstantRange& pcRange = layout->getPushConstantRanges().begin()[0];
```

Comment:
you cannot rely on this, ask the user for the offset of the swapchain orientation push constants (you know the size and stage flag)